helm-charts/infisical/templates/backend-deployment.yaml

```diff
@@ -34,32 +35,24 @@ spec:
           periodSeconds: 10
         ports: 
         - containerPort: 4000
-        {{- if .Values.backend.kubeSecretRef }}
+        {{- if $backend.kubeSecretRef }}
         envFrom:
         - secretRef:
-            name: {{ .Values.backend.kubeSecretRef }}
+            name: {{ $backend.kubeSecretRef }}
         {{- end }}
         env:
         - name: MONGO_URL
           value: {{ include "infisical.mongodb.connectionString" . | quote }}
-        {{- if .Values.backendEnvironmentVariables }}
-        {{- range $key, $value := .Values.backendEnvironmentVariables }}
```

Comment:
is there a reason why you deleted the abality to write envs right into the deployment file? I'm gussing because it is more secure and can be stored in `kubeSecretRef`? Or were you thinking or something else

Reply:
Oh i see, you put them in a secret below

Reply:
I agree, this is better approach

Reply:
I think the variable `backendEnvironmentVariables` was completely removed. This will break existing installations.

Reply:
well, it's up to @maidul98 if you want to keep this non breaking or not. I just wanted to point it out 😅